draft-ietf-quic-qpack.md

```diff
@@ -477,28 +479,27 @@ it can choose to use a lower dynamic table capacity (see
 
 For clients using 0-RTT data in HTTP/3, the server's maximum table capacity is
 the remembered value of the setting, or zero if the value was not previously
-sent.  When the client's 0-RTT value of the SETTING is 0, the server MAY set it
```

Comment:
To me, zero looks better written in numbers, especially with that many occurrences of "non-zero" in the same section.

Reply:
Just for the record, I do not feel strongly, and I'm happy to incorporate whatever @afrind suggests.

Reply:
Yeah, I guess zero pairs better with non-zero?  I don't think it makes a huge difference either way.

Reply:
Thanks for chiming in.  In that case I'm keeping this change.